src/services/study/network-map.js

```diff
@@ -137,3 +146,93 @@ export function fetchAllCountries(studyUuid, currentNodeUuid) {
     console.debug(fetchCountriesUrl);
     return backendFetchJson(fetchCountriesUrl);
 }
+
+function createEquipmentIdentifierList(equipmentType, equipmentList) {
+    return {
+        type: 'IDENTIFIER_LIST',
+        equipmentType: equipmentType,
+        filterEquipmentsAttributes: equipmentList.map((eqId) => {
+            if (eqId?.id) {
```

Comment:
Why this test ?

Reply:
The list of equipment. Each equipment can be either a string representing the equipment ID or an object with an 'id' property.

i added  comments to explay the arguments